docs/issue_triage_guide.md

```diff
@@ -0,0 +1,35 @@
+# Issue Triage Guide
+
+This guide describes how to triage GitHub issues on the [issue queue](https://github.com/openservicemesh/osm/issues) for Open Service Mesh (OSM).
+
+## Steps for Triage
+1. Identify the cause of the issue as explained [below](#identifying-the-cause-of-the-issue)
+1. Tag the issue with relevant tags
+1. Prioritize the issue with a P1, P2 or P3 label where:
+    - P1: product cannot ship. Should be addressed as soon as possible. Eg: high priority bug
+    - P2: Product cannot ship, but it does not need to be addressed immediately. Eg: high priority feature, lower priority bug
+    - P3: Resolution of the work item is optional based on resources, time, and risk. Eg: a nice to have feature
+1. Size the issue and assign it to a milestone:
+    [WIP]
+1. If you don't have time to resolve it, put links to the code to where you see the problem arising
```

Comment:
`If you have time and can resolve the issue, resolve it. If not, put links [...]